content.md

```diff
@@ -1,125 +1,126 @@
-
-Skills
-------
-
-#### Expert in:
-- Languages (computer): **Python**, **Bash** (sad, but true)
-- Languages (human): **English** (written and spoken!)
-- Large-scale system management:
-  - **Large-Scale System Design**
-  - **Monitoring/alerting**
-  - **Reliable deployment**
-- **Linux** system administration
-
-#### Enthusiastic dilettante in:
-- Languages (computer): **JavaScript/ECMAScript**, **Node.js**, **Go**
-- Languages (human): **Spanish**
-- **Teaching** peers and young students.
-
-#### It’s been a while, but I could pick it back up:
-- Languages (computer): **C**, **C++**, **Java**, **Ruby** (on or off Rails), **Perl**, **Lisp**
-- Languages (human): **French**, **Hindi**, **Bengali**
-
-
-Education
----------
-
-### University of Virginia, Dept of Computer Science
-#### M.Cs. in Computer Science - Jan 2008
-Thesis project: **“Color CT: Capturing Energy Dependence in CT Scans”**  
-Publications: Williams, Daniel, et al. **"A cross-layer approach to heterogeneity and reliability."** *Proceedings of the 7th IEEE/ACM international conference on Formal Methods and Models for Codesign.* IEEE Press, 2009.
-
-### University of Virginia, School of Engineering & Applied Science
-#### B.S. with Honors in Computer Science - May 2005<br/> Also completed requirements for B.A. in Cognitive Science
-Thesis: **“NJTopol: Adding Topology to Parallelized Neural Simulation”**  
-Publications: Levy, William B., et al. **"The formation of neural codes in the hippocampus: trace conditioning as a prototypical paradigm for studying the random recoding hypothesis."** *Biological cybernetics* 92.6 (2005): 409-426.
+# Aprotim Sanyal 
+ 
+jobs@aprotim.com   
+(310) 243-6230 
+ 
+Employment 
+------------ 
+### Edgecast - **Principal Software Engineer, SRE** - *April 2020 - July 2022* 
+ 
+Rather than focusing on a single service, as a member of a small SRE task force for the entire organization, I identified areas of cross-cutting concern where we could act as a catalyst to amplify effort across other engineering teams and led several of these efforts. 
+ 
+- **Built organization-wide best practices.** My primary responsibility as an experienced SRE was evaluating existing procedures and tools 
+  across EdgeCast to create consistent processes and tooling in line with best practices. 
+  - **Evangelized SRE best practices to leadership.** Educated senior directors and VPs on the value of **SLIs/SLOs, error budgets, 
+    consistent tooling and practices**, etc. to build a better data-driven platform. 
+  - **Established SLIs and SLOs across teams.** Having established the value of SL\[IO\]s, I took 
+    on the large task of meeting with service teams across the org to understand their services and dependencies. I explained what makes useful, actionable SLIs, and coached team representatives through writing and instrumenting new SLIs and SLOs in line with these standards. 
```

Comment:
Led company-wide initiative with service teams to understand responsibilities and dependencies, and provide training to cross-functional engineering teams on establishing actionable SLIs. Extensively coached teams on writing and instrumenting new SLIs and SLOs in line with improved standards